start_pokealarm.py

```diff
@@ -140,9 +141,26 @@ def parse_settings(root_path):
                         help='Maximum number of attempts an alarm makes to send a notification.')
     parser.add_argument('-tz', '--timezone', type=str, action='append', default=[None],
                         help='Timezone used for notifications.  Ex: "America/Los_Angeles"')
+    parser.add_argument('--log_to_file',
+                        help=('Enable logging to file.'),
+                        action='store_true', default=False)
+    parser.add_argument('--log_path',
+                        help=('Defines directory to save log files to.'),
+                        default='logs/')
 
     args = parser.parse_args()
 
+    if args.log_to_file:
+        # Create directory for log files.
+        if not os.path.exists(args.log_path):
+            os.mkdir(args.log_path)
+        date = strftime('%Y%m%d_%H%M')
+        filename = os.path.join(args.log_path, '{}.log'.format(date))
```

Comment:
Any particular reason you decided to log to a new file each time instead of just appending?

Reply:
To be honest I simply copied over the functionality from RocketMap (2 commandline params + new file for each launch).